src/transformers/models/detr/configuration_detr.py

```diff
@@ -43,6 +44,11 @@ class DetrConfig(PretrainedConfig):
     documentation from [`PretrainedConfig`] for more information.
 
     Args:
+        use_timm_backbone (`bool`, *optional*, defaults to `True`):
+            Whether or not to use the `timm` library for the backbone. If set to `False`, will use the [`AutoBackbone`]
+            API.
+        backbone_config (`PretrainedConfig` or `dict`, *optional*, defaults to `ResNetConfig()`):
```

Comment:
The default is only when `use_timm_backbone=False` so would remove it from here and put it in the doc below.
```suggestion
        backbone_config (`PretrainedConfig` or `dict`, *optional*):
```